frontend/main.js

```diff
@@ -149,10 +150,10 @@ async function startApp () {
           errorNotification('handleEvent', e, message)
         }
       },
-      processError: (e: Error, message: GIMessage) => {
+      processError: (e: Error, message: GIMessage, msgMeta) => {
```

Comment:
What type is this `msgMeta`? I think we should create a type for it.

Reply:
Done